src/pages/tutorial/angular/step-1.mdx

```diff
@@ -0,0 +1,905 @@
+---
+title: 1. Installing Carbon
+tabs:
+  ['Overview', 'Step 1', 'Step 2', 'Step 3', 'Step 4', 'Step 5', 'Wrapping up']
+---
+
+import Preview from 'components/Preview';
+
+### Starting with the Carbon Angular, there are two ways to begin working with Carbon components. By the end, these two methods will produce the same result.
+
+<AnchorLinks>
+
+<AnchorLink>Prerequisites</AnchorLink>
+<AnchorLink>Use our starter app - recommended</AnchorLink>
```

Comment:
I can't because then the href becomes `"#use-starter-app-(recommended)"` whereas the title has the id `use-starter-app-recommended` even with the brackets.

Reply:
Could we use a `a` directly? Like `<a href="#use-starter-app-recommended">`